### PR TITLE
Cleanup unity

### PR DESCRIPTION
**Pull request by nlferman**
Just a bunch of cleanup of obsolete things, and a bit of polish to the User Settings panel. Plus some other small stuff.

### CODE REVIEW

File: AR-Top-MobileApp/Assets/Prefabs/MapController.cs [hunk 1]
```diff
@@ -141,7 +139,7 @@ void buildLayers ()
 	{
 	
 		mapLayer = new GameObject ("MapLayer");
-		mapLayer.AddComponent <LeanRotate> ();
+		mapLayer.AddComponent <LeanRotate1> ();
```
Comment (joshherkness) [inline AR-Top-MobileApp/Assets/Prefabs/MapController.cs]: Can we not think of a better name for this?

File: AR-Top-MobileApp/Assets/Prefabs/MapController.cs [hunk 2]
```diff
@@ -71,7 +69,7 @@ void Start ()
 		string dataAsJson = "";
 
 		// Load example file
-		string filePath = Path.Combine(Application.streamingAssetsPath, "TestData/voxel_shell_8x8.json");
+		string filePath = Path.Combine(Application.streamingAssetsPath, "TestData/voxel_cube_8x8.json");
```
Comment (joshherkness) [inline AR-Top-MobileApp/Assets/Prefabs/MapController.cs]: This change can probably be removed.

File: AR-Top-MobileApp/Assets/UserSettings.cs [hunk 3]
```diff
@@ -37,12 +33,30 @@ public class UserSettings : MonoBehaviour {
...
+		setDropdownValue ();
+	}
+
+	void setDropdownValue (){
+		int index = 2;
+		if (PlayerPrefs.HasKey ("UserAA")) {
+			index = PlayerPrefs.GetInt ("UserAA");
 		}
+		TMP_Dropdown dropdown = dropdowns [0];
+		switch (index) {
```
Comment (joshherkness) [inline AR-Top-MobileApp/Assets/UserSettings.cs]: This switch case can probably be changed to a simple mathematical formula, we have an integer value that gets converted to another integer value? Just a thought.

**Review by joshherkness — CHANGES_REQUESTED** [review]

File: AR-Top-MobileApp/Assets/Prefabs/MapController.cs (hunk 1 above)
Comment (nlferman) [reply AR-Top-MobileApp/Assets/Prefabs/MapController.cs]: Changed it to LeanRotateYAxis

File: AR-Top-MobileApp/Assets/Prefabs/MapController.cs (hunk 2 above)
Comment (nlferman) [reply AR-Top-MobileApp/Assets/Prefabs/MapController.cs]: This does have some uses for testing purposes but I commented it out.

File: AR-Top-MobileApp/Assets/UserSettings.cs (hunk 3 above)
Comment (nlferman) [reply AR-Top-MobileApp/Assets/UserSettings.cs]: Changed it to a bitshift. Also changed the anti-aliasing setter to a powering function.

**Comment by nlferman** [comment]
I made the requested changes, let me know what you think.

**Review by joshherkness — APPROVED** [review]

**Comment by joshherkness** [comment]
Looks good. If you ever want to tack changes onto the previous commit, you can use `git commit --amend`

**Pull request merged**